### ISSUE TITLE
nem2-cli does not encrypt the profiles

### ISSUE THREAD

**Issue by dgarcia360** [nemtech/nem2-cli#26]
nem2-cli stores the accounts in plain text. It should handle private keys securely, display warning messages before overwriting an account, and allow to create backups.

**Comment by BuBuHighLighter** [comment]
@dgarcia360 
What should I do to handle private keys securely?
I ask the user to set a password to decrypt the profile?

**Comment by dgarcia360** [comment]
The protocol to store private keys security should be shared with [the desktop wallet](https://github.com/nemfoundation/nem2-desktop-wallet) and the mobile wallet. We can continue the conversation on [public slack #sig-client],((https://app.slack.com/client/T9EAN12HX/CDBKDV6CU/thread/DH1JA49UK-1568166461.003100).)  see how other wallet devs are doing and try to consensuate a standard. 

In NIS1, we used an encrypted .wlt file. The wallet file was read every time an account announced a transaction, after typing its password. For Catapult, we have the class [Wallet](https://nemtech.github.io/nem2-sdk-typescript-javascript/classes/_model_wallet_wallet_.wallet.html) that we might use, but we could evaluate other proposals to enhance security and usability.

At the moment, I propose to keep this issue "on hold" until there is consensus on how wallets manage private keys.

**Comment by dgarcia360** [comment]
@evias We'll need some guidance to unblock this issue based on how other client wallets are saving wallets.

**Comment by dgarcia360** [comment]
New issue #153
